kalite/distributed/features/steps/superuser_create.py

```diff
@@ -5,6 +5,10 @@
 
 modal_container = "superusercreate-container"
 
+@given("I go to homepage")
```

Comment:
I intentionally avoid using the @given("I am on the homepage") because it will create a super user which I don't want.